quality/lint.xml

```diff
@@ -11,6 +11,9 @@
     <ignore path="src/main/res/drawable/ic_patient_male.xml" />
     <ignore path="src/main/res/drawable/ic_patient_transgender.xml" />
 
+    <!-- TODO: Remove once the layout is used in the future PR -->
+    <ignore path="src/main/res/layout/view_allpatientsinfacility.xml" />
```

Comment:
Let's clean up `lint.xml`? All the resources mentioned above this line seem to be used somewhere.

Reply:
I *think* this will need to be removed in the next PR when the view actually gets embedded in the parent screens. I'll remove this and confirm.

Reply:
Tested just now, can't remove it in this PR. Will remove in the final PR.